Client/src/Features/Auth/authSlice.js

```diff
@@ -74,6 +75,28 @@ export const update = createAsyncThunk(
   }
 );
 
+export const deleteUser = createAsyncThunk(
+  "auth/delete",
+  async (data, thunkApi) => {
+    const user = jwtDecode(data);
+
+    //1.5s delay to show loading spinner
+    await new Promise((resolve) => setTimeout(resolve, 1500));
```

Comment:
Is this just for demonstration purposes?  I don't think we need to slow down operations purposefully when we're ready to ship production

Reply:
Yea pretty much, without the timeout the operation is instant. The timeout is there to tell the user _hey, a process is occurring_. I would image that with larger datasets there won't be a need for it at all.